tests/integration/nm/gen_conf_test.py

```diff
@@ -82,6 +84,51 @@ def test_gen_conf_ovs_same_name(eth1_up, cleanup_ovs_same_name):
     retry_verify_ovs_ports("br0", sorted(["eth1", "br0"]))
 
 
+@pytest.mark.tier1
+def test_gen_conf_routes_rules():
+    desired_state = load_yaml(
+        """
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv4:
+    address:
+      - ip: 192.0.2.251
+        prefix-length: 24
+    dhcp: false
+    enabled: true
+routes:
+  config:
+    - destination: 198.51.100.0/24
+      metric: 150
+      next-hop-address: 192.0.2.1
+      next-hop-interface: eth1
+      table-id: 254
+route-rules:
+  config:
+    - ip-to: 192.0.2.0/24
+      ip-from: 192.168.2.0/24
+      priority: 1
+      route-table: 254
+"""
+    )
+    result = libnmstate.generate_configurations(desired_state)[
+        "NetworkManager"
+    ]
+    with nm_service_restart():
```

Comment:
Please use `testlib.gen_conf.gen_conf_apply()`

Reply:
Done, thank you!